pom.xml

```diff
@@ -32,163 +41,40 @@
         <jenkins.version>2.32.3</jenkins.version>
         <pipeline.version>2.5</pipeline.version>
         <cps.global.lib.version>2.6</cps.global.lib.version>
-        <workflow-cps-plugin.version>2.28</workflow-cps-plugin.version>
     </properties>
 
-
     <dependencies>
-
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-aggregator</artifactId>
             <version>${pipeline.version}</version>
-        </dependency>
+        </dependency>        
```

Comment:
[meta] how strict should we be with whitespace changes?

Reply:
I think since it is a clean up for this time it is ok

Reply:
Basically I do not agree with the statement above. Cleanups are allowed to be less strict than other changes wrt white spaces is not valid IMO. I personally think we should avoid white space changes since it blurs the focus of the commit.

Reply:
ok, done